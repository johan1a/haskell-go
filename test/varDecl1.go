var x int = 2