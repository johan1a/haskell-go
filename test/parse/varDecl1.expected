SourceFile (Package "main") [TopLevelDecl1 (VarDecl [IdDecl "x"] (TypeName (TypeNameIdentifier "int")) [Num 2])]
